plugins/module_utils/network/sonic/config/logging/logging.py

```diff
@@ -393,6 +418,10 @@ def get_delete_requests(self, configs, delete_all):
             if servers_request:
                 requests.extend(servers_request)
 
+        if ('security_profile' in configs or delete_all) and configs['security_profile'] != "":
```

Comment:
Delete can be performed, only when there is an existing configuration.
```suggestion
        if configs.get('security_profile'):
```

Reply:
There is a case for handling all delete using same proc